### DISCUSSION TITLE
Slight difference in resolution between orthomosaic and labeled image.

### DISCUSSION THREAD

**Discussion by Jordan-Pierce** [cnr-isti-vclab/TagLab#70] [Q&A]
Hi All, 

I noticed a slight difference in resolution (dimensions, height x width) between the orthomosaic and the labeled image output by TagLab as a PNG. The difference is rather small, but I was curious about why this might be? Does anyone know if TagLab tries to stick to a specific ratio or am I missing something very obvious? 👀

![image](https://user-images.githubusercontent.com/115024024/200429950-f84332a2-0d91-4d8c-9ebd-293cf0517287.png)

**Comment by Jordan-Pierce** [comment]
I tried again, but this time included a DEM along with the orthomosaic. Both were set to a specific set of dimensions, and when labels were output from TagLab as both PNG and TIF, the dimensions matched the input. Might just a ratio resizing issue, or perhaps the DEM needs to be included?

**Reply by maxcorsini** [reply]
Hi Jordan,

the DEM is not necessary. Regarding the difference in resolution sounds a bit strange. Can you send us your project, or a toy project where the problem happens, so that we can check ? Thank you.
